.github/workflows/wheels.yml

```diff
@@ -66,3 +66,49 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
           retention-days: 7
+
+  build_ios:
+    name: Build wheels for iOS
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+        with:
+          submodules: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v5.5.0
+        with:
+          python-version: "3.x"
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install -U pip
+          # Use main branch until iOS support is released.
+          # python -m pip install cibuildwheel==3.0.0
+          python -m pip install git+https://github.com/pypa/cibuildwheel.git
+
+      - name: Build wheels
+        run: python -m cibuildwheel
+        env:
```

Comment:
I think you can set `PYMUNK_BUILD_SLIM: 1` here, just like done on wasm/Pypodide. It will exclude the examples from the wheel which if I understand it correctly are not useful anyway, or are they?